aws-redshift-cluster/src/main/java/software/amazon/redshift/cluster/ReadHandler.java

```diff
@@ -56,6 +59,13 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
                     .build();
         }
 
+        /*
+        NAMESPACE_RESOURCE_POLICY_ACTION will be true if NamespaceResourcePolicy property is included in the template.
+        This attribute will be used to decide if "not authorized to perform: redshift:GetResourcePolicy" errors
+        in Read handler should be suppressed or not.
+         */
+        NAMESPACE_RESOURCE_POLICY_ACTION = model.getNamespaceResourcePolicy() != null;
```

Comment:
can we update this to containsResourcePolicy? and let's not use all uppercase since this is not a static variable

Reply:
Done.